README.rst

```diff
@@ -40,6 +40,7 @@ Installation
 
     pip install loguru
 
+Or if you need more secure defaults, install ``loguru-hardened``.
```

Comment:
I also mentioned `loguru-hardened` in the README, but I could not find a release action configured, so I'm not sure how to make sure hardened is also released when a new release is created.

Reply:
Indeed, since there is no Github Action, I need to do it manually for now.

Reply:
I could open up another pr with release workflows if you would like that. I don't know what currently goes into the manual release actions, but it might be a nice setup to simplify maintenance.

Reply:
@bneijt Thanks for the proposal. If you're familiar with release automation via Github Actions, your help is most welcome. I'd like to replace manual release management.

Also sorry for not having yet merged this PR, but it seems it will be incompatible with https://github.com/Delgan/loguru/pull/1140. I need to check how to create `loguru-hardened` without `setup.py`.

Reply:
just use a Python script change project.name in new pyproject.toml file

Reply:
i can help write a ci file

Reply:
@trim21 Yes, I suppose this should be straightforward. Unfortunately, this PR will need to be updated, though.

Reply:
consider merge 1140 first maybe....